backend/src/main/java/com/example/backend/use_case/predictions/PredictionsInteractor.java

```diff
@@ -0,0 +1,15 @@
+package com.example.backend.use_case.predictions;
+
+public class PredictionsInteractor implements PredictionsInputBoundary {
+    final PredictionsOutputBoundary outputPresenter;
+    final PredictionsDataAccessInterface dataAccess;
```

Comment:
I would change the name of the object to be more descriptive, specifically adding the "predictions" part. 

Btw, there is no PredictionsDataAccessInterface. Which is a larger issue than naming.

Reply:
Ok just noticed that the DAI exists, but its in the wrong directory. Needs to be moved to the proper directory.

Reply:
I would not worry about this right now, since it is not crucial to running the backend for this issue. We can fix this later.

Reply:
I resolved the problem.